### DISCUSSION TITLE
Fix dead links

### DISCUSSION THREAD

**Discussion by sebastian-92** [3kh0/ext-remover#1167] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I compiled a large list of links, to be displayed on my site, however a bunch of them are dead. How do I get rid of the dead links? I have over 1900, so it needs to be automatic and large scale

**Comment by s0urce-c0de** [comment, marked as answer]
I could help, but is it GET requests? What is it?
You could try:
```python3
import urllib2 as A
with open(input('What file are all the urls stored in?'))as C:
 for B in C.read().split('\n'):
  try:A.urlopen(B);print(B)
  except A.HTTPError:...
```
if the file is like
```
link1
https://thisis.anotdead.link/
DEDLINK
```
I think it might work, but best to add http or https to the start of all the url.

**Reply by sebastian-92** [reply]
I think this will work thanks

**Reply by s0urce-c0de** [reply]
your welcome. i did this on the spot, so i have no idea if it actually works.
